Add audit events for desktop clipboard access

zmb3 opened the pull request:
Introduce two new audit events:

- desktop.clipboard.send: emitted when a user's local clipboard
  data is sent to teleport
- desktop.clipboard.receive: emitted when clipboard data is received
  from a remote windows desktop

Closes #9706

zmb3 committed:
Add audit events for desktop clipboard access

Introduce two new audit events:

- desktop.clipboard.send: emitted when a user's local clipboard
  data is sent to teleport
- desktop.clipboard.receive: emitted when clipboard data is received
  from a remote windows desktop

Closes #9706

ibeckermayer reviewed (approved):
I believe we need to add client side support for these as well: https://github.com/gravitational/webapps/issues/602

rosstimothy commented on api/types/events/events.proto:
```diff
@@ -247,6 +247,48 @@ message DesktopRecording {
+    int32 Length = 6 [ (gogoproto.jsontag) = "length" ];
+}
+
+// DesktopClipboardSend is emitted when when clipboard data is
```
```suggestion
// DesktopClipboardSend is emitted when clipboard data is
```

zmb3 committed:
Only request CF_OEMTEXT clipboard data

For now, we only support the most basic type of data.
If we request more than one type, we end up calling into Go more
than once, which generates duplicate audit events and sends the
data to the browser multiple times.
